README.md

```diff
@@ -3,13 +3,18 @@
 A collection of helpful utilities for use in Dart projects. Right now, it
 includes the following:
 
-  * `Disposable` interface / mixin to assist with cleaning up streams and other
-  data structures that won't necessarily be garbage collected without some
+  * A `Cache` implementation that maintains references to an object instance by
+  an identifier. Specializations of this class allow invariants to be
+  maintained:
+    * A `ReferenceCache` maintains a count for each access for a given identifier.
+    When the final reference is released, the item is removed from the cache.
```

Comment:
This renders a little awkwardly and probably not how you intended. It might be nice if this was indented in the rendered markup. This is how it looks now:

![image](https://cloud.githubusercontent.com/assets/8091340/21726017/1ac50aa2-d3f8-11e6-8a49-5b3268026d5f.png)